yogstation/code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm

```diff
@@ -154,6 +154,19 @@ Difficulty: Insanely Hard
 	melee_damage_upper = 20
 	move_to_delay = 3
 
+/mob/living/simple_animal/hostile/retaliate/goat/guard/pope
+	name = "Goat Pope"
+	desc = "For what is a god without a pope to spread it's holy words"
```

Comment:
>God
>It
bruh.

Reply:
Archbishop of Canterbury said god is gender neutral so shush